ansible/bootstrap.yml

```
@@ -1,5 +1,20 @@
 ---
 
+- hosts: ansible_init
```

```suggestion
- hosts: cluster
```

Lets do this so we ALWAYS check, regardless of how inventory groups are set up. And it's not `all` in case localhost is in all.